working-docs/slack-channel-introduction.md

```diff
@@ -0,0 +1,44 @@
+
+# New Here? Walkthrough
+New to the Diversity & Inclusion slack channel?
+
+Here’s a couple of things you should know before you dive into the channel!
+
+## What is DD&I
+
+Drupal Diversity and Inclusion is an independent collective of volunteers who advocate for greater diversity in the community and more inclusive events, help create safe spaces, and build the capacity of allies.
+
+This group was created after DrupalCon New Orleans. Nikki Stevens & Karyn Cassio gave a presentation about diversity and inclusion. ++(link presentation page)++
```

Comment:
Hmm maybe this is a conversation on the older commit version. The link is there now thank you!